bundles/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/internal/ControllerHandler.java

```diff
@@ -146,25 +151,19 @@ public boolean isConnected() {
                 logger.trace("Sending command '{}'", command);
                 OutputStream out = localSocket.getOutputStream();
                 out.write(command.getBytes());
```

Comment:
Shouldn't the character encoding be specified here as well?

Reply:
Original code didn't, so I didn't do this either. But I have fixed this and will push a commit.